src/FSharp.AWS.DynamoDB/Expression/ConditionalExpr.fs

```diff
@@ -455,54 +483,96 @@ let writeConditionExpression (writer : AttributeWriter) (cond : ConditionalExpre
 
     let rec writeQuery q =
         match q with
-        | False | True -> invalidOp "internal error: invalid query representation"
-        | Not q -> ! "( NOT " ; writeQuery q ; ! " )"
-        | And (l,r) -> ! "( " ; writeQuery l ; ! " AND " ; writeQuery r ; ! " )"
-        | Or (l,r) -> ! "( " ; writeQuery l ; ! " OR " ; writeQuery r ; ! " )"
-        | Compare (cmp, l, r) -> ! "( " ; writeOp l ; writeCmp cmp ; writeOp r ; ! " )"
-        | Between (v,l,u) -> ! "( " ; writeOp v ; ! " BETWEEN " ; writeOp l ; ! " AND " ; writeOp u ; ! " )"
-        | BeginsWith (attr, op) -> ! "( begins_with ( " ; !(writer.WriteAttibute attr) ; ! ", " ;  writeOp op ; !" ))"
-        | Contains (attr, op) -> ! "( contains ( " ; !(writer.WriteAttibute attr) ; !", " ; writeOp op ; !" ))"
-        | Attribute_Exists attr -> ! "( attribute_exists ( " ; !(writer.WriteAttibute attr) ; ! "))"
-        | Attribute_Not_Exists attr -> ! "( attribute_not_exists ( " ; !(writer.WriteAttibute attr) ; ! "))"
-        | In(op,ops) -> ! "(" ; writeOp op ; ! " IN (" ; writeOps ops ; ! "))"
+        | False
+        | True -> invalidOp "internal error: invalid query representation"
+        | Not q ->
+            ! "( NOT "
```

Comment:
this sort of stuff is just bad news - maybe if there was a `prettier ignore` annotation you could att at function level?

Reply:
Pretty sure you can only ignore at file level

Reply:
maybe thats the thing to do as the bulk of files do seem mostly fine overall

Reply:
seems cured, will delete this thread after

Reply:
wait, not sure why I thought it was cured, leaving comment here